packages/portal-subgraph/subgraph.yaml

```diff
@@ -0,0 +1,42 @@
+specVersion: 0.0.4
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum
+    name: Portal
+    network: ropsten
+    source:
+      address: "0xD7A15a7B9c09E5126aB4598601dF5935C7Ab4Bec"
```

Comment:
do we need to specify the address both here and in `networks.json`?

Reply:
got rid of the hardcode, templatized using mustache